plugins/obs-webrtc/whip-service.cpp

```diff
@@ -1,7 +1,11 @@
 #include "whip-service.h"
 
 const char *audio_codecs[MAX_CODECS] = {"opus"};
+#ifdef ENABLE_HEVC
 const char *video_codecs[MAX_CODECS] = {"h264", "hevc", "av1"};
+#else
+const char *video_codecs[MAX_CODECS] = {"h264", "av1"};
+#endif
```

Comment:
We decided in #10500 to not guard enums with ENABLE_HEVC because it made other checks overly complex. Our main concern is that HEVC related code cannot execute. Is there any similar risk here of causing overly complex checks by guarding this value here?

Reply:
Thinking more on this, this is _probably_ fine, because it is only used to set the supported codecs for the WHIP service. Unlike the rtmp_common services which provide their supported codecs in services.json, we have to provide the supported codecs list in code here.